rfc6962-bis.xml

```diff
@@ -1515,6 +1545,26 @@ target="http://csrc.nist.gov/publications/fips/fips180-4/fips-180-4.pdf">
         <format type="PDF" target="https://cabforum.org/wp-content/uploads/EV_Certificate_Guidelines.pdf"/>
       </reference>
 
+      <reference anchor="Chromium.Policy" target="http://www.chromium.org/Home/chromium-security/certificate-transparency">
+	<front>
+	  <title>Chromium Certificate Transparency</title>
+	  <author>
+	    <organization>The Chromium Projects</organization>
+	  </author>
+	  <date year="2014"/>
+	</front>
+      </reference>
+
+      <reference anchor="JSON.Metadata" target="http://www.certificate-transparency.org/known-logs/log_list_schema.json">
+	<front>
+	  <title>Chromium Log Metadata JSON Schema</title>
+	  <author>
+	    <organization>The Chromium Projects</organization>
```

Comment:
Not that it matters much, but is the log list truly owned by the Chromium project? it uses it but that's it..